src/components/ModelSelector.jsx

```diff
@@ -41,66 +31,25 @@ export default function ModelSelector({ toggleControls }) {
         <Col className="text-center">
           <Button onClick={toggleControls}>Options</Button>
         </Col>
-        <Col className="text-center">
-          <ToggleButtonGroup
-            type="radio"
-            name="measurement"
-            value={selection.measurement}
-            onChange={(val) => handleChange(val)}
-          >
-            {measurement.map((m) => {
-              return (
-                <ToggleButton key={m.name} value={m}>
-                  {m.name}
-                </ToggleButton>
-              );
-            })}
-          </ToggleButtonGroup>
-        </Col>
 
-        <Col className="text-center">
-          <ToggleButtonGroup
-            type="radio"
-            name="season"
-            value={selection.season}
-            onChange={(val) =>
-              selectorDispatch({
-                type: "TOGGLE_SEASON",
-                payload: val,
-              })
-            }
-          >
-            {season.map((m) => {
-              return (
-                <ToggleButton key={m.name} value={m}>
-                  {m.name}
-                </ToggleButton>
-              );
-            })}
-          </ToggleButtonGroup>
-        </Col>
-
-        <Col className="text-center">
-          <ToggleButtonGroup
-            type="radio"
-            name="tide"
-            value={selection.tide}
-            onChange={(val) =>
-              selectorDispatch({
-                type: "TOGGLE_TIDE",
-                payload: val,
-              })
-            }
-          >
-            {tide.map((m) => {
-              return (
-                <ToggleButton key={m.name} value={m}>
-                  {m.name}
-                </ToggleButton>
-              );
-            })}
-          </ToggleButtonGroup>
-        </Col>
+        {Object.keys(BUTTONS).map((buttonGroup) => (
```

Comment:
```suggestion
        {Object.entries(BUTTONS).map(([group, options]) => (
```

or similar.  This saves the need for a second access to `BUTTONS` on line 43